src/types/index.ts

```diff
@@ -155,10 +155,13 @@ export interface SAMLUser {
 export type PortalUser = {
   userId: string
   mySpace: Collection[]
+  displayName: string
 }
 
 export type SessionUser = SAMLUser & {
   userId: string
+  givenname?: string
+  surname?: string
```

Comment:
Suggestion: It looks like `SAMLUser` has `givenname` and `surname` in the type under `attributes`. It doesn't look like we use `givenname` or `surname` directly off of `SAMLUser` as far as I can tell. So we should be able to delete these two lines.